bot/utils/blurple_formatter.py

```diff
@@ -0,0 +1,717 @@
+import ast
+import random
+from collections.abc import Generator
+from random import randrange
+
+max_line_length = 50
+
+op_strs = {
+    ast.Add: "+",
+    ast.Mult: "*",
+    ast.Sub: "-",
+    ast.Div: "/",
+    ast.Mod: "%",
+    ast.Pow: "**",
+    ast.BitXor: "^",
+    ast.BitOr: "|",
+    ast.BitAnd: "&",
+    ast.LShift: "<<",
+    ast.RShift: ">>",
+    ast.Invert: "~",
+    ast.Not: "not",
+    ast.UAdd: "+",
+    ast.USub: "-",
+    ast.Eq: "==",
+    ast.NotEq: "!=",
+    ast.Lt: "<",
+    ast.LtE: "<=",
+    ast.Gt: ">",
+    ast.GtE: ">=",
+    ast.Is: "is",
+    ast.IsNot: "is not",
+    ast.In: "in",
+    ast.NotIn: "not in",
+    ast.And: "and",
+    ast.Or: "or",
+}
+
+# operator precedence table
```

Comment:
```suggestion
# Operator precedence table
```

...you get the point, if you change this one capitalize the others as well!

Reply:
resolved in 8c224a52be5000af380465a9d71a55997750fccd